extra/1-radio-stations.js

```diff
@@ -14,7 +14,11 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies() {
+  const frequencies = [87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108]
+  return frequencies ;
```

Comment:
You should do this using a for loop. It's simple enough to hardcode the array here as it's quite short, but imagine if you had an array with hundreds or thousands of values -  a for loop would be your only practical option.